### PR TITLE
VIH-9736 SpecFlow - Manage team

### PR DESCRIPTION
**Pull request by hrishi-moj**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/VIH-9736
https://tools.hmcts.net/jira/browse/VIH-9737

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by will-craig — DISMISSED** [review]

File: UI/Selenium/Pages/ManageWorkAllocationPage.cs [hunk 1]
```diff
@@ -40,28 +41,86 @@ public class ManageWorkAllocationPage
...
         public static By SetEndDate = By.Id("end-date");
         public static By FilterDate = By.Id("filter-btn");
 
         // Manage team
-        public static By ManageTeam = By.Id("manage-team");
-        public static By ManageTeamSearchTeamMemberField1 = By.Id("search-team-member");
-        public static By ManageTeamSearchTeamMemberField = By.Id("username");
+      public static By ManageTeam = By.Id("manage-team");
+    
+     // public static By ManageTeamSearchTeamMemberField1 = By.Id("search-team-member");
```
Comment (shaed-parkar) [inline UI/Selenium/Pages/ManageWorkAllocationPage.cs:51]: could you remove commented code please?

File: UI/Selenium/Steps/WorkAllocationManageTeamEditUser.cs [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+            Driver.FindElement(ManageWorkAllocationPage.ManageTeamEditRoleConfirmationMsg).Text;
+        Assert.AreEqual(ManageTeamEditRoleConfirmationExpected, ManageTeamEditRoleConfirmationActual);
+    }
+
+    [Then(@"I change the Role from CSO to Administrator")]
+    public void ThenIChangeTheRoleFromCsoToAdministrator()
+    {
+        ExtensionMethods.WaitForElementVisible(Driver, ManageWorkAllocationPage.ManageTeamJusticeUserEditRole);
+        Driver.FindElement(ManageWorkAllocationPage.ManageTeamJusticeUserEditRole).Click();
+        // ExtensionMethods.WaitForElementVisible(Driver, ManageWorkAllocationPage.ManageTeamJusticeUserRoleAdministration);
```
Comment (shaed-parkar) [inline UI/Selenium/Steps/WorkAllocationManageTeamEditUser.cs:114]: Commented code for removal?

**Review by marcogagliardi — DISMISSED** [review]
👌🏽

**Review by phildebrah — DISMISSED** [review]

**Comment by hrishi-moj** [comment]
New Branch created

**Pull request closed without merging**